### PR TITLE
Update benchmark scores

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Update BM scores after one repo was closed

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark
- [x] TODO: rollback custom BM fork after https://github.com/Samsung/CredData/pull/63

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        repository: babenek/CredData
```
Comment (babenek) [inline .github/workflows/benchmark.yml:25]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:65]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -166,7 +166,7 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:169]:
```suggestion
        repository: Samsung/CredData
```

**Review by babenek — COMMENTED** [review]
prepared for rollback custom BM fork

**Review by iuriimet — APPROVED** [review]

**Review by Yullia — APPROVED** [review]
Approved

**Pull request merged**